src/Components/RosTable/ExpandedRow.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import {
+    TextContent,
+    TextList,
+    TextListVariants,
+    TextListItem,
+    TextListItemVariants
+} from '@patternfly/react-core';
+
+export const ExpandedRow = ({ id, cloudProvider, instanceType, idlingTime, ioWait }) =>  {
+    const inlineFieldsArray = [
+        { label: 'Provider', value: cloudProvider },
+        { label: 'Instance Type', value: instanceType },
+        { label: 'Idling time', value: idlingTime },
+        { label: 'I/O Wait', value: ioWait }
+    ];
+
+    return (
+        <React.Fragment>
```

Comment:
Just a nitpick, no need to wrap in `React.Fragment` if you have just one child. But you can keep as is.